Marsiple/Common/Network/MartianApiManager.swift

```diff
@@ -49,4 +56,28 @@ private extension DataFetcher {
         }
         task.resume()
     }
+    
+    static func postData(data: Data, url: URL) {
```

Comment:
Why doesn't this method have `success` and `failure` callbacks? How does the caller know anything about the result of the operation?

Also, we could also make this a lot more flexible by specifying the `data` argument to be `Codable` and doing the `JSON` -> `Data` encoding within this method.

Reply:
Apologies for the lack of clarification. The `CommentViewController` should be poped of the stack only if the operation `successfully`. 

If not, it should display an alert saying that `shit has hit the fan`.